src/render/shape.es.frag

```diff
@@ -0,0 +1,10 @@
+#version 300 es
+precision highp float;
+
+in vec4 v_color;
+
+out vec4 o_Target;
+
+void main() {
+    o_Target = v_color;
```

Comment:
bevy_webgl2 currently requires explicit conversion to SRGB colorspace, like:

```
vec4 encodeSRGB(vec4 linearRGB_in)
{
    vec3 linearRGB = linearRGB_in.rgb;
    vec3 a = 12.92 * linearRGB;
    vec3 b = 1.055 * pow(linearRGB, vec3(1.0 / 2.4)) - 0.055;
    vec3 c = step(vec3(0.0031308), linearRGB);
    return vec4(mix(a, b, c), linearRGB_in.a);
}

void main() {
    o_Target = encodeSRGB(v_color);
}
```
In the future this will be done automatically by postprocessing pass, but for now we have to deal with it this way.